### DISCUSSION TITLE
Python bot_local.py can not run

### DISCUSSION THREAD

**Discussion by vincentvaldinata** [teamalgoritma/community#371] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76637079/193369874-83f646c4-30a2-4ce4-9d1c-cadc913cbdf3.png)

i already installed python extension on my visual studio code but still can not run

**Comment by danteistanto** [comment, marked as answer]
selamat pagi, bantu jawab ka, sy kemarin dibantu ka Kevin @kevbow .
https://github.com/teamalgoritma/community/discussions/364#discussioncomment-3769313

semoga membantu

**Reply by vincentvaldinata** [reply]
thank you
